src/MongoDB.Driver.Core/MongoDB.Driver.Core.targets

```diff
@@ -1,15 +1,19 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
+    <PropertyGroup>
+      <CompressionRuntimesPath>runtimes/win-x64/native</CompressionRuntimesPath>
+    </PropertyGroup>
+
     <ItemGroup Condition="$([MSBuild]::IsOsPlatform('Windows'))">
-        <Content Include="$(MSBuildThisFileDirectory)../x64/native/windows/snappy64.dll">
+        <Content Include="$(MSBuildThisFileDirectory)../$(CompressionRuntimesPath)/snappy64.dll">
```

Comment:
Here, I've just changed the path in line 5: https://github.com/DmitryLukyanov/mongo-csharp-driver/pull/104/files#diff-309a27f3df928fd0cd771cbbc6f0e933L5. All other logic is the same as before.
**NOTE**s:
1. This file is used only for **.net framework** logic
2. It's possible to achieve copying the whole `runtimes` folder(before we copied only the native library) into `bin` for that case (as well as I did for net core 3.0 in this PR). But see two points why I haven't done it:

- I didn't find a way to remove `Core` folder from the user's solution for **.net framework** applications. So, currently, all users application will have this folder:
![image](https://user-images.githubusercontent.com/44870738/73772201-c5f12780-4790-11ea-8e36-bce0bf65d1cd.png)
and if so, this folder should not be useless :) 
- But even if the above point looks solvable, there is an issue with `.net core 2.0` where I didn't find a way to copy any nuget data into the `bin` folder. It looks like it's expected behavior for .net core2.0. So, to use nugets, applications will have only application libraries inside `bin` folder, all nugets data will be used from global nugets storage (%USER%/.nuget/packages). 
It means that we won't be able to get rid of the relative path for native libraries `..\..\` here: https://github.com/DmitryLukyanov/mongo-csharp-driver/pull/104/files#diff-44d116f9a7cf8bf722f905bd6199cdefR56 in any case

Reply:
This relative path  `..\..\` is also used to get native libraries from the Core folder (see the above screenshot) for **.net framework**.